doc/tutorial/sequence_file/sequence_file_snippets.cpp

```diff
@@ -1,5 +1,9 @@
 #include <fstream>
 #include <numeric> // std::accumulate
+//![include_ranges]
+#include <seqan3/std/ranges>
+//![include_ranges]
+
 //![include_ranges_chunk]
 #include <range/v3/view/chunk.hpp>
```

Comment:
```suggestion
#include <seqan3/utility/views/chunk.hpp>
```

Reply:
I fix that also in the rest of the header xD This PR now turned into https://github.com/seqan/product_backlog/issues/124